web/shared/src/fenced_code.ts

```diff
@@ -1,6 +1,21 @@
 import katex from "katex";
 import _ from "lodash";
+import assert from "minimalistic-assert";
 
+type GeneratedPygmentsData = {
```

Comment:
I think `PygmentsData` might be a better name; the fact that it's "generated" isn't important to the frontend codebase.